package.json

```diff
@@ -32,7 +32,8 @@
     "svgs:illustrations": "svgr --config-file svgr.config.js -d src/shared/illustrations src/shared/illustrations/svgs",
     "svgs": "yarn svgs:icons && yarn svgs:illustrations",
     "graphql": "graphql-codegen --config codegen.config.yml",
-    "svgs:import": "node ./scripts/figma-import"
+    "svgs:icons-import": "node ./scripts/figma-import",
+    "svgs:icons-import-generate": "svgs:icons-import && svgs:icons"
```

Comment:
I suggest we flip the names here - `svgs:icons` could be the group action (import + generate) and then we'd have `svgs:icons-import` for import and `svgs:icons-generate` to run svgr